RecoMuon/Configuration/python/RecoMuon_EventContent_cff.py

```diff
@@ -55,7 +55,9 @@
                                            'keep TrackingRecHitsOwned_refittedStandAloneMuons_*_*',
                                            'keep recoTracks_displacedStandAloneMuons__*',
                                            'keep recoTrackExtras_displacedStandAloneMuons_*_*',
-                                           'keep TrackingRecHitsOwned_displacedStandAloneMuons_*_*'
+                                           'keep TrackingRecHitsOwned_displacedStandAloneMuons_*_*',
+                                           # Sim matching information
+                                           'keep *_muonSimClassifier_*_*'
```

Comment:
This should be moved to a sim-specific place.
e.g. define RecoSimMuon{AOD,RECO,FEVT} and add it to Configuration/EventContent/python/EventContent_cff.py in the corresponding SIM tiers.

Reply:
fixed